docarray/document/mixins/image.py

```diff
@@ -50,33 +50,41 @@ def convert_blob_to_image_tensor(
         return self
 
     def convert_image_tensor_to_uri(
-        self: 'T', channel_axis: int = -1, image_format: str = 'png'
+        self: 'T',
+        channel_axis: int = -1,
+        image_format: str = 'png',
+        quality: int = 90,
     ) -> 'T':
         """Assuming :attr:`.tensor` is a _valid_ image, set :attr:`uri` accordingly
 
         :param channel_axis: the axis id of the color channel, ``-1`` indicates the color channel info at the last axis
         :param image_format: either `png` or `jpeg`
+        :param quality: quality of the image in case of using jpeg
```

Comment:
According to https://pillow.readthedocs.io/en/stable/handbook/image-file-formats.html?#jpeg
> The image quality, on a scale from 0 (worst) to 95 (best). The default is 75. Values above 95 should be avoided; 100 disables portions of the JPEG compression algorithm, and results in large files with hardly any gain in image quality.

Can we also use the default 75 quality, and also add explanation for quality in docstr